### DISCUSSION TITLE
Include package name in subject line on PyPI reports

### DISCUSSION THREAD

**Discussion by shenanigansd** [vipyrsec/bot#41] [General]
Actually include the name of the package in the subject line of package reports so we're not sending multiple emails that just say "automated report".

**Comment by shenanigansd** [comment]
Do we really need the subject to be customizable?
Can we not just make it always be the package name?
For example -
```diff
- subject = discord.ui.TextInput(
-   label="Subject",
-   placeholder="Subject",
-   default="Automated PyPi Malware Report",
-   required=True,
-   style=discord.TextStyle.short,
- )
```
```diff
- subject=self.subject.value,
+ subject=f"PyPi Malware Report: {self.package.title}",
```

If we wanted to be able to customize the subject, we could also have the input be added as a suffix after the default name.

**Comment by import-pandas-as-numpy** [comment]
The last 100+ reports I've sent contain the title "Malicious Package Report". 
I don't think it's a major issue, they manually review each report regardless. 
That said, I don't think it hurts to have the package in the e-mail title, so if it's trivial to implement, I don't see a compelling reason why we shouldn't.
